frontend/packages/core/src/Table/table.tsx

```diff
@@ -79,13 +79,14 @@ export const Table: React.FC<TableProps> = ({ headings, children, ...props }) =>
 
 export interface TableRowProps extends Pick<MuiTableRowProps, "onClick"> {
   children?: React.ReactNode;
+  defaultValue?: any;
```

Comment:
We should probably have this be more specific to start with and make it broader later if needed. Let's use `React.ReactNode` as the type as this will allow a react element or a string